### DISCUSSION TITLE
reverse proxy + npm [Jul 16]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2822] [Discord Threads]
> 🙋‍♂️ **sph**

**Comment by o1lab** [comment]
> 🙋‍♂️ **sph**  
#### hmm, it does work with portforward over ssh  
#  
> 🙋‍♂️ **sph**  
#### ~~managed to get it working with `proxy_pass http://10.23.2.5:9001/noco/`~~   
####    
#### api urls won't work this way :/  
#  
> 🙋‍♂️ **sph**  
#### ```nginx   
####   location ^~ /noco/dashboard {   
####     if ($scheme != "https") {   
####       return 301 https://$server_name$request_uri;   
####     }   
####     proxy_set_header X-Forwarded-For $remote_addr;   
####     proxy_set_header X-Forwarded-Proto 'https';   
####     proxy_set_header Host $host;   
####     proxy_pass http://10.23.2.5:9001/;   
####    
####     rewrite /noco(.*) $1 break;   
####   }   
####   location ^~ /noco/ {   
####     if ($scheme != "https") {   
####       return 301 https://$server_name$request_uri;   
####     }   
####     proxy_set_header X-Forwarded-For $remote_addr;   
####     proxy_set_header X-Forwarded-Proto 'https';   
####     proxy_set_header Host $host;   
####     proxy_pass http://10.23.2.5:9001/;   
####   }   
#### ```   
#### that's one unholy piece of kludge but it works   
#### `NC_DASHBOARD_URL=/dashboard`  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/997602123799859231)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
